test/test-template-engine.js

```diff
@@ -1476,28 +1451,10 @@ describe('rendering errors', () => {
 
 
   describe('unsafe', () => {
-    it('throws if used on an unexpected language', () => {
```

Comment:
We don’t support `unsafe` svg for now. We may choose to add this back in the future in some backwards-compatible way.